Gemfile

```diff
@@ -6,6 +6,6 @@ gem "nokogiri"
 gem "standard"
 
 group :jekyll_plugins do
-  gem 'pry'
-  gem 'jekyll_frontmatter_tests'
+  gem "pry"
```

Comment:
This is the first time I've looked at a PR in this repo since actions were added, but do you think it would be possible to squash this commit into the relevant earlier commits where these files were added?

Reply:
I've had a look at other PRs and it looks like automatic fixes are a thing with their own commits, so you can ignore me!